website/markdown/docs/plugins/using-plugins.mdx

```diff
@@ -0,0 +1,58 @@
+---
+name: Using plugins
+excerpt: Learn how to extend Tuist with plugins.
+---
+
+# Using Plugins
+
+## Plugins
+
+With plugins you can extend Tuist to add support for custom project description helpers, custom templates, and much more!
+Plugins are meant to be separate isolated directories that contain custom code or templates used to extend existing Tuist functionality. 
+You may use plugins that exist locally on your machine or better yet, upload them to the internet and keep them under source control.
```

Comment:
```suggestion
Plugins are meant to be separate and isolated directories, each containing custom code or templates used to extend existing Tuist functionality. 
You may use plugins that exist at a path local to your machine or at a git source URL to enable sharing and version control across multiple projects.
```

It could also be in source control local your project, not necessarily an either/or, so might be better to just document local vs git. That would give a bit more space to talk about tag vs sha and be explicit about what the options are.

Also unclear: are these paths relative to the project? Absolute?